### ISSUE TITLE
App is not restarted on error

### ISSUE THREAD

**Issue by Somebi** [pilu/fresh#75]
By the looks of it, you don't listen for signals of process you are executing after build.
If there is some error afterwards, it doesn't do anything.
Listen for signals and restart app or exit pilu/fresh on app error to allow docker container to restart itself.

It's critical if used inside docker.

**Comment by gmccue** [comment]
Could you explain a bit more what exactly your issue is?

**Comment by DiegoGallegos4** [comment]
I agree on this, even trying to make a syntactic error, fresh won't notice it and still build. That would be the easiest reproducible bug.

**Comment by denisrudov** [comment]
when exit(1)
it doesn't reflect on fresh. understandable because it watches the files and nothing were changed
and I have to restart the whole service inside docker-compose :( 

@gmccue 
> Could you explain a bit more what exactly your issue is?
